### DISCUSSION TITLE
I don't want documentation but

### DISCUSSION THREAD

**Discussion by ape1121** [Sixze/ALS-Refactored#120] [Q&A] [answered]
Can you please tell us how to add overlay event blueprints?

**Comment by Sixze** [comment, marked as answer]
1. Create a new gameplay tag for your overlay. You can add it via project settings. This is optional, but if you want your tag to be displayed inside the Q menu, then add it to the `Als.OverlayMode` tag group.
2. Create an animation blueprint for your overlay. You can duplicate one of the existing ones and just delete its contents.
3. Open `BP_Als_Character` and add your overlay tag and animation blueprint to the `OverlayAnimationInstanceClasses` map inside the Class Defaults tab.

That's all.

**Reply by ape1121** [reply]
Thank you for your answer 🙏 And where would you suggest me to put overlay-specific input actions? In the character bp?

**Reply by Sixze** [reply]
It's up to you. Animation blueprints can't handle input, so they are not an option.

**Reply by ape1121** [reply]
For example, where is the aim input handled?

**Reply by Sixze** [reply]
In `AAlsCharacterExample`.

**Reply by ape1121** [reply]
Thank you very much.
